src/utils/validateNotes.ts

```diff
@@ -0,0 +1,14 @@
+import { t } from '@i18n'
+
+export const noFirstSymbolsPattern = /^[!"#$%&'()*+,./:;<=>?@[\\\]^_`{|}~§±№-]/
+
+export const validateNotes = (
+  note: string,
+  setError: (arg: string) => void,
```

Comment:
Validate functions are not supposed to do side effects. They should return either the value is valid or not.
In components it might be used like this:
`if (isNotesValid(notes)) {
  // ...do stuff
} else {
  setError('Notes are not valid!')
}`